.ci/travis/travis.sh

```diff
@@ -461,6 +461,33 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-error-test-josm)
+  echo "installing new ant ..."
+  wget http://www-us.apache.org/dist//ant/binaries/apache-ant-1.10.7-bin.zip
+  unzip -q apache-ant-1.10.7-bin.zip
+  export ANT_HOME=$(pwd)/apache-ant-1.10.7
+  export PATH=$ANT_HOME/bin/:$PATH
+  echo "building checkstyle ..."
+  mvn -e clean package -Passembly
+  export CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  echo "Checkouting sources ...."
+  svn -q export https://josm.openstreetmap.de/svn/trunk/ josm
+  cd josm
+  sed -i -E "s/(name=\"checkstyle\" rev=\")([0-9]+\.[0-9]+(-SNAPSHOT)?)/\1${CS_POM_VERSION}/" \
+   tools/ivy.xml
+  ant -v checkstyle
+  ls -la
+  cat checkstyle-josm.xml
```

Comment:
please  remove this `cat` to make logs huge, we do not need this.

```
$ git status
On branch don-vip/enable-josm
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   .ci/travis/travis.sh

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	apache-ant-1.10.7-bin.zip
	apache-ant-1.10.7/
	josm/

no changes added to commit (use "git add" and/or "git commit -a")
```

please clone/download all to `.ci-temp` folder, `git status`  should be clean.

Reply:
@nmancus1 , please remind me where we stand on `git status` validation in CIs ?
do we have hanging PR for this ?

Reply:
No hanging PR, Shippable and Wercker are done, I just need to finish work on Travis, which I will complete this week.

Reply:
@don-vip ,

there is still a problem:
```
$ ./.ci/travis/travis.sh no-error-test-josm
....
$ git status
On branch don-vip/enable-josm
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	apache-ant-1.10.7-bin.zip
	apache-ant-1.10.7/
```
please download to `.ci-temp` folder. `git status` should be empty.

Reply:
@nmancus1 , your issue is about cleanup of `.ci-temp`, it is not the same.
this problems is more critical, I moved `git status` validation to https://github.com/checkstyle/checkstyle/issues/8228 .